internal/handler/command.go

```diff
@@ -498,8 +594,17 @@ func createCommandValueFromDR(dr *contract.DeviceResource, v string) (*dsModels.
 	return result, err
 }
 
-func CommandAllHandler(cmd string, body string, method string, queryParams string) ([]*dsModels.Event, common.AppError) {
-	common.LoggingClient.Debug(fmt.Sprintf("Handler - CommandAll: execute the %s command %s from all operational devices", method, cmd))
+func CommandAllHandler(
+	cmd string,
+	body string,
+	method string,
+	queryParams string,
+	req *http.Request) ([]*dsModels.Event, common.AppError) {
```

Comment:
See my other comment regarding reducing the amount of arguments since some can be derived from others.